src/main/java/pvs/app/controller/ProjectController.java

```diff
@@ -133,4 +133,20 @@ public ProjectController(ProjectService projectService, RepositoryService reposi
         //         3        //
         //////////\\\\\\\\\\\\
     }
+
+    @PostMapping("/project/remove/{projectId}")
+    public ResponseEntity<String> removeProject(@PathVariable Long projectId) {
+        if (projectService.removeProjectById(projectId)) {
+            return ResponseEntity.status(HttpStatus.OK).body(successMessage);
+        } else {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(urlInvalidMessage);
+        }
```

Comment:
Please unwrap the `else ... ` when it has already returned above.
```suggestion
        }
        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(urlInvalidMessage);
```

Reply:
Why not use `else {return ...}`? Is it practical or better?